test/unit/stream-id-stack-tests.js

```diff
@@ -62,8 +63,8 @@ describe('StreamIdStack', function () {
     stack.clear();
   });
   it('should release unused groups', function (done) {
-    var releaseDelay = 50;
-    var stack = newInstance(3, releaseDelay - 10);
+    var releaseDelay = 100;
```

Comment:
I question myself if is it better to use a fixed value or a relative (percentage)